BugsnagNetworkRequestPlugin/BugsnagNetworkRequestPlugin.xcodeproj/project.pbxproj

```diff
@@ -1141,7 +1149,7 @@
 				TVOS_DEPLOYMENT_TARGET = 9.2;
 				VERSIONING_SYSTEM = "apple-generic";
 				VERSION_INFO_PREFIX = "";
-				WATCHOS_DEPLOYMENT_TARGET = 4.0;
+				WATCHOS_DEPLOYMENT_TARGET = 6.0;
```

Comment:
Aren't we targeting 2.0?

Reply:
Yes, but this plugin won't work without `NSURLSessionTaskMetrics`, which requires watchos 6.

Reply:
But the podspec says `2.0`

https://github.com/bugsnag/bugsnag-cocoa/blob/watchos-network-plugin-scheme/BugsnagNetworkRequestPlugin.podspec.json#L22

This would mean Cocoapods would happily add the BugsnagNetworkRequestPlugin but there'd then be a link error.

The iOS deployment target is set to 9.0 but the logic only kicks in on iOS 10 and above... this is the same pattern we should be following here.

Reply:
(this makes it easier for devs to integrate into their projects)

Reply:
I've updated the podspec to 6.0